### PR TITLE
Add `number` in the list of properties supported for rules

### LINKED ISSUE #1079: include current PR but not other open PRs
We are using this action to automatically populate a change log file, that is then merged back to main once PR is merged.

This allows to approve also the generated change log as part of the PR (which may be convenient to spot that a bad naming is used in the PR).

To do so, we use the `includeOpen` option, but this option includes as well any other Open PR. Is there a way to include only the current open PR?

Thx

### PR DESCRIPTION
**Pull request by chicco785**
fixes #1079

### CODE REVIEW

**Review by mikepenz — APPROVED** [review]
Thank you very much for your contribution!

File: __tests__/transform.test.ts [hunk 1]
```diff
@@ -528,14 +565,53 @@ it('Use Rules to get all open PRs in a Category.', async () => {
...
+          on_property: 'status'
         }
       ]
     }
   ]
+  expect(buildChangelogTest(customConfig, prs)).toStrictEqual(`## Open PRs only\n\n- Still pending open pull request\n   - PR: #6\n\n`)
+})
+
+
+it('Use Rules to get current open PR and merged categorised.', async () => {
```
Comment (chicco785) [inline __tests__/transform.test.ts:578]: @mikepenz thanks for your support!

I have the feeling I am doing something wrong.... to check if the rules works correctly, i added a test. it seems that rules are combined with `and` logic and due to that, no output is generated in this case.

**Comment by mikepenz** [comment]
Thank you very much for the contribution. Decided to rename `exclusive` again to `exhaustive` to keep it in line with the current naming scheme

**Comment by chicco785** [comment]
> Thank you very much for the contribution. Decided to rename `exclusive` again to `exhaustive` to keep it in line with the current naming scheme

Thank you for accepting my contribute!

**Pull request merged**